tests/openshift/templates/README.md

```diff
@@ -0,0 +1,68 @@
+# BuildConfig
+
+## Description
+
+These buildconfigs can be used to create a fresh imagestream, which can be used in a Jenkinsfile.
+
+## Usage
+
+Process this file, creating or replacing imagestreams and builds. Make sure to specify a project.
+
+```
+oc process -f openshift/<..>.bc.yaml | oc [create|replace] -n <PROJECT> -f -
+```
+
+Consume this image in a Jenkinsfile. This example uses a declarative syntax.
+
+```
+stage('Functional Tests') {
+    steps {
+        script {
+            openshift.withCluster() {
+                openshift.withProject(TOOLS_PROJECT) {
+                    echo "Functional Testing"
+                    podTemplate(
+                        label: "bddstack-${env.CHANGE_ID}",
+                        name: "bddstack-${env.CHANGE_ID}",
+                        serviceAccount: 'jenkins',
+                        cloud: 'openshift',
+                        containers: [
+                            containerTemplate(
+                                name: 'jnlp',
+                                image: 'docker-registry.default.svc:5000/<PROJECT>/bddstack:latest',
+                                resourceRequestCpu: '800m',
+                                resourceLimitCpu: '800m',
+                                resourceRequestMemory: '4Gi',
+                                resourceLimitMemory: '4Gi',
```

Comment:
We should probably review these resource limits at some point in the future, even if this is just a recommendation example.